### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by paulkefer**
Begins addressing #957 for German. Will continue force-pushing until complete.

### CODE REVIEW

File: app/Locale/de_DE/LC_MESSAGES/cake_console.po [hunk 1]
```diff
@@ -14,244 +14,244 @@ msgstr ""
 "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
 msgid "Comparing Database to Schema..."
-msgstr ""
+msgstr "Vergleiche Datenbasis mit Schema..."
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/cake_console.po]: Better translation would be `Datenbank`

File: app/Locale/de_DE/LC_MESSAGES/cake_console.po [hunk 2]
```diff
@@ -14,244 +14,244 @@ msgstr ""
...
 
 msgid "Schema is up to date."
-msgstr ""
+msgstr "Schema ist aktuell."
 
 msgid "The following statements will run."
-msgstr ""
+msgstr "Die folgenden statements werden ausgeführt."
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/cake_console.po]: `statements` needs to be `Statements` with an uppercase S

File: app/Locale/de_DE/LC_MESSAGES/cake_console.po [hunk 3]
```diff
@@ -14,244 +14,244 @@ msgstr ""
...
-msgstr ""
+msgstr "<error>%s Fehler:</error> %s\n"
 
 msgid " <comment>(optional)</comment>"
-msgstr ""
+msgstr " <comment>(optional)</comment>"
 
 msgid " <comment>(choices: %s)</comment>"
-msgstr ""
+msgstr " <comment>(optionen: %s)</comment>"
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/cake_console.po]: Uppercase `Optionen`

File: app/Locale/de_DE/LC_MESSAGES/default.po [hunk 4]
```diff
@@ -15,292 +15,292 @@ msgstr ""
...
-msgstr ""
+msgstr "Anfrage ge-black-holed"
 
 msgid "Debug setting does not allow access to this url."
-msgstr ""
+msgstr "Debug-Einstellung erlaubt keinen Zugriff auf diese url."
 
 msgid "Cache engine %s is not available."
-msgstr ""
+msgstr "Cache engine %s nicht verfügbar."
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/default.po]: Cache-Engine

File: app/Locale/de_DE/LC_MESSAGES/default.po [hunk 5]
```diff
@@ -15,292 +15,292 @@ msgstr ""
...
 
 msgid "Cache engines must use %s as a base class."
-msgstr ""
+msgstr "Cache engines müssen %s als Basisklasse verwenden."
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/default.po]: Cache-Engine

File: app/Locale/de_DE/LC_MESSAGES/default.po [hunk 6]
```diff
@@ -15,292 +15,292 @@ msgstr ""
...
-msgstr ""
+msgstr "%s cache konnte '%s' nicht ins %s cache schreiben"
 
 msgid "Invalid cache group %s"
-msgstr ""
+msgstr "Ungültige cache-Gruppe %s"
 
 msgid "Files cannot be atomically decremented."
-msgstr ""
+msgstr "Dateien können nicht atomisch vermindert werden."
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/default.po]: automatisch

File: app/Locale/de_DE/LC_MESSAGES/default.po [hunk 7]
```diff
@@ -15,292 +15,292 @@ msgstr ""
...
 
 msgid "Files cannot be atomically incremented."
-msgstr ""
+msgstr "Datein können nicht atomisch erhöht werden."
```
Comment (nook24) [inline app/Locale/de_DE/LC_MESSAGES/default.po]: automatisch

File: app/Locale/de_DE/LC_MESSAGES/default.po (hunk 6 above)
Comment (paulkefer) [reply app/Locale/de_DE/LC_MESSAGES/default.po]: Wouldn't the English word for 'automatisch' be 'automatically'? It says 'atomically', which I would translate as 'atomar' or 'atomisch'. I believe this refers to the atomical decrement, e.g. `$var--`

File: app/Locale/de_DE/LC_MESSAGES/default.po (hunk 7 above)
Comment (paulkefer) [reply app/Locale/de_DE/LC_MESSAGES/default.po]: As above

**Pull request merged**